_internal/DeerLister.php

```diff
@@ -179,25 +179,33 @@ private function isHidden(string $path, bool $ignoreHide = false): bool
 
         if (array_key_exists("forbidden", $config) && $config["forbidden"] !== NULL)
         {
-            $hidden = $config["forbidden"];
+            $forbidden = $config["forbidden"];
         }
         else
         {
-            $hidden = ["_internal", "vendor"];
+            $forbidden = ["_internal", "vendor"];
         }
         if (!$ignoreHide && array_key_exists("hidden", $config) && $config["hidden"] !== NULL)
         {
-            $hidden = [...$hidden, ...$config["hidden"]];
+            $hidden = $config["hidden"];
```

Comment:
`$hidden` will be undefined when ignoring hide, throws warning

Reply:
What do you mean?

Reply:
`$hidden` is undefined when `$ignoreHide` is true, that will throw below since you are passing to to the foreach

Reply:
Ah yeah right